README.md

```diff
@@ -18,6 +18,34 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
 
 # Tips list
 
+## {{tip no}} - Passing arguments to callback functions
+> {{date}} by [@minhazav](https://twitter.com/minhazav)
+
+By default you cannot pass arguments to a callback function. For eaxmple:
+```js
+function callback() {
+	console.log('Hi human');
+}
+
+$('#someelem').on('click', callback);
+```
+You can take advantage of the closure scope in Javascript to pass arguments to callback functions. Check this example:
+```js
+function callback(a, b) {
```

Comment:
another way to pass arguments into callback functions is by using bind - http://codepen.io/tevko/pen/NxvaPP?editors=001